apps/opencs/view/render/lighting.cpp

```diff
@@ -1,5 +1,38 @@
 #include "lighting.hpp"
 
 #include <osg/LightSource>
+#include <osg/NodeVisitor>
+#include <osg/Switch>
+
+#include <components/misc/constants.hpp>
+
+class SwitchWindowsGlowVisitor : public osg::NodeVisitor
```

Comment:
Can this be more generic? I assume Glowing is something that happens with not just Windows but perhaps with other meshes/statics? For example a Cave entrance, or an object underwater?

Reply:
You are right, but I am not very good with inventing names on English.